src/Symfony/Component/Security/Guard/AbstractGuardAuthenticator.php

```diff
@@ -36,9 +36,6 @@ public function supports(Request $request)
      * Shortcut to create a PostAuthenticationGuardToken for you, if you don't really
      * care about which authenticated token you're using.
      *
-     * @param UserInterface $user
-     * @param string        $providerKey
```

Comment:
I‘d keep `@param string $providerKey` as long as we don’t have the type-hint here.

Reply:
Agreed, @thomasbisignani Can you revert the removal of `string` params?

Reply:
Done. Thx !